### PR TITLE
Emily Webster Homework 1

### PR DESCRIPTION
**Pull request by elwebster**
For this assignment, I added a title, headers for the short summary and longer summary, a short set of bullet point summaries for the biography, then added a longer biography. The longer biography includes two images, one of snorkelling at the Great Barrier Reef and one of visiting a temple in Jaipur. I also included a link to a website, called Environmental History Now.

I found the markdown basics guide on the RStudio website really helpful for learning the syntax of R Markdown (https://rmarkdown.rstudio.com/authoring_basics.html). I encountered some problems inserting the images -- or rather, I thought I was encountering problems inserting the images, as I tried to run the command caption as R code and it wouldn't run. But then I realized that the reason it wasn't running was likely because it was syntax for Markdown, but not strictly R code. When I had R Markdown knit the file together, the images appeared fine. So that was a lot of scouring websites looking for the reasons for the error (unexpected '[' in "!["), but at least I learned how I could alternatively code an image if needed. I also had to change the file pathway for the images from a file path specific to my computer and not relative, so set it up to be relative (I asked a friend who is more familiar with coding about this particular problem, and I changed it accordingly), and changed an image name to remove spaces so it would load properly in Github.

Before submitting your pull request, have you:

* Included all relevant files in your fork (e.g. R Markdown files, rendered Markdown files, images/graphs)?
* Added or updated a relevant `README.md` file describing the purpose of your repository, required packages, location of relevant files, etc.?

Be sure to include your name somewhere in the title or description of your pull request, as well as reflect on what was hard/easy, problems you solved, helpful tutorials you read, etc.

### CODE REVIEW

**Review by surreinm — COMMENTED** [review]
Hi Emily. I really liked that you included a tl;dr which makes good use of the bullet feature in r markdown though if I had to make a suggestion I would say to add a photo of the mentioned cat and dog. The pictures that you did include aren't showing up though. I believe the issue is due to the fact that the image isn't in the correct place in your files. I don't think that GitHub can reach your dropbox unfortunately. Here's a [link](https://www.earthdatascience.org/courses/earth-analytics/document-your-science/add-images-to-rmarkdown-report/) that might be helpful to add pictures in the future.

File: README.md [hunk 1]
```diff
@@ -1 +1,34 @@
...
+* Dog and cat mom
+* Taking CFSS to beef up computing skills for epidemiological analysis/mapping
+
+### Actual Summary
+
+Hello! My name is Emily, and I am PhD Candidate in the history department here at UChicago. I work on historical epidemics of microbial disease (specifically plague, tuberculosis, and typhoid fever) across the British Empire. Because I am such a disease nerd, I am also working on an MSc in Public Health Science, so I can get a handle on epidemiological modeling and analysis. 
+
+I just returned from 18 months of dissertation research in the UK, India, and Australia, which led to some very cool moments, like these:
+
+![](./Great_Barrier_Reef.png)
```
Comment (kaitlynvanbaalen) [inline README.md:17]: This is so cool! The pictures integrated very well into your ReadMe, and I like that you captioned them in italics.

File: README.md [hunk 2]
```diff
@@ -1 +1,34 @@
...
+
+
+
+
+![](./Jaipur.png)
+
+*Visiting Birla Mandir in Jaipur*
+
+
+Outside of work, I love to hike and hang out with my dog and cat. I also edit a feminist environmental history blog called [Environmental History Now](https://envhistnow.com/). We try to showcase the work of female-identifying early career scholars across environmental studies. It was started by a BU student named Elizabeth Hameeteman, and has grown into something pretty remarkable - I recommend checking it out!
```
Comment (kaitlynvanbaalen) [inline README.md:29]: Nice inclusion of a link. Overall this looks very professional and informative.

**Comment by jyumanzor** [comment]
Hi Emily. I love the detail of your intro and how you used pictures and a link to know more about you.  I also think it was clever to use the format of tl;dr to incorporate markdown. I couldn't see the pictures but perhaps I'm not viewing the right version.

File: README.md (hunk 2 above)
Comment (jyumanzor) [reply README.md:29]: Hi Emily. I love the detail of your intro and how you used pictures and a link to know more about you. I also think it was clever to use the format of tl;dr to incorporate markdown. I couldn't see the pictures but perhaps I'm not viewing the right version.

**Review by kwabene — COMMENTED** [review]
Hi Emily! Overall, your submission looks very clean. As far as I can tell, you used an example of all five of the elements we were supposed to practice in Markdown, so that's great. Also, it looks like you used all of these elements in ways that made sense and did not seem forced at all. Great job.

File: README.md (hunk 1 above)
Comment (bensoltoff) [reply README.md:17]: Good method of embedding the image if it is not already hosted elsewhere online

**Review by bensoltoff — COMMENTED** [review]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Improvement |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |           |       x       |            |
| **Coding strategy**         |           |        x      |            |
| **Presentation**            |       x    |              |            |
| **Achievement, creativity** |   x        |              |            |
| **Reproducibility**         |     x      |              |            |

**Review by jessezorr — COMMENTED** [review]
Hi Emily,

I liked your summary a lot; it seems like you did everything that the assignment asked for.  I particularly liked your pictures: I couldn't figure out how to get mine to behave and shrink to a reasonable size as you did. Your coding looks good! Also, funnily enough, I've spend nearly a year in Jaipur - small world!

I also like your tl;dr summary of the summary.  Great work!

**Pull request closed without merging**